src/lavinmq/shovel/shovel.cr

```diff
@@ -359,7 +360,20 @@ module LavinMQ
                else
                  "/"
                end
-        response = c.post(path, headers: headers, body: msg.body_io)
+        
+        # Compute signature if secret is configured
+        # Note: This requires reading the entire body into memory to compute HMAC.
+        # For webhook use cases, messages are typically small event notifications.
+        if secret = @signature_secret
+          body = msg.body_io.gets_to_end
+          signature = OpenSSL::HMAC.hexdigest(OpenSSL::Algorithm::SHA256, secret, body)
+          headers["X-LavinMQ-Signature-256"] = "sha256=#{signature}"
```

Comment:
Why is it called "LavinMQ-Signature-256"? it's nothing "lavinmq" about it, and sha256 is already in the value, why not just `X-HMAC-Signature`?

Reply:
Ok, looking on other implementations of this I see that there's no real standard, hence the product name in the header, and then you have to look up the docs for that header. Well, then either `256` in the key or the `sha256=` in the value should at least go away. 

I also liked that [slack](https://docs.slack.dev/authentication/verifying-requests-from-slack/) included the date/time in the signature to prevent reply attacks. Have we considered that? (But instead of adding a custom time/date header, why not just use the standard [`Date`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Date) header?)

Reply:
Key rotation is also not supported, as [PagerDuty](https://developer.pagerduty.com/docs/verifying-webhook-signatures) does